_run/metallb-service.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
```

Comment:
is this in `_run/` because the `_docs/kustomize` versions won't work with our `kind` environment?

how about adding an explanation to `_run/README.md` for these?

Reply:
I put it here because it is at this point only used to support the `_run/` examples. But I don't see any reason why this wouldn't work in a real provider on mainnet. @sacreman do we have a recommended way to add an optional component to provider deployments like this?